lib/pliny/tasks/db.rake

```diff
@@ -34,7 +34,7 @@ begin
     task :seed do
       if File.exist?('./db/seeds.rb')
         database_urls.each do |database_url|
-          Sequel.connect(database_url)
+          DB = Sequel.connect(database_url)
```

Comment:
I fail to see the negative consequences of this, other than the constant being set after the `each` is done. My understanding here is that for each database, it'll set the `DB` constant before loading the seeds. Am I missing something else?